spring-boot-jib/pom.xml

```diff
@@ -0,0 +1,68 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-boot-jib</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>spring-boot-jib</name>
+    <packaging>war</packaging>
+
+    <parent>
+        <artifactId>parent-boot-2</artifactId>
+        <groupId>com.baeldung</groupId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../parent-boot-2</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
So, there are some test dependencies here, but no tests.

First, do you think we are okay here without tests? The app is simple enough that I'm a bit impartial.

Second, if we aren't doing tests, let's drop the test dependencies.

In fact, let's make the pom as minimal as possible, just to keep the focus on the tutorial's topic.

Reply:
Sure. I don't think we need tests here, since the focus is on configuration anyway. I will remove the test dependencies

Reply:
Removed the test dependencies